exasol/analytics/schema/column.py

```diff
@@ -1,76 +1,461 @@
-import dataclasses
-from typing import Optional
+# do not from __future__ import annotations
```

Comment:
Should I maybe split the large file column.py into
* `column/__init__.py`
* `column/boolean.py`
* `column/char.py`
* `column/date.py`
* `column/decimal.py`
* `column/double.py`
* `column/geometry.py`
* `column/hashtype.py`
* `column/timestamp.py`
* `column/varchar.py`